docs/src/rules/generator-star.md

```diff
@@ -126,3 +131,7 @@ If your project will not be using generators you do not need this rule.
 ## Further Reading
 
 * [Understanding ES6: Generators](https://leanpub.com/understandinges6/read/#leanpub-auto-generators)
+
+## Resources
+
+* [Documentation source](https://github.com/eslint/eslint/tree/HEAD/docs/src/rules/generator-star.md)
```

Comment:
Why are we manually adding this section to removed rules? It will be a duplicate of the auto-generated one (note "Resources" both before and after "Version"):

https://deploy-preview-933--eslint.netlify.app/docs/rules/generator-star

Reply:
This just didn’t get stripped with the automation I was using to apply changes. Should be removed here.